### ISSUE TITLE
Minor typo/bug in enabling training mode for model_fine

### ISSUE THREAD

**Issue by SelvamArul** [krrish94/nerf-pytorch#21]
In this [line](https://github.com/krrish94/nerf-pytorch/blob/a14357da6cada433d28bf11a45c7bcaace76c06e/train_nerf.py#L171),
shouldn't it be `model_fine.train()` instead of `model_coarse.train()`?

**Comment by krrish94** [comment]
@SelvamArul That is correct -- I believe there is a PR pending merge that fixes this. I'll get to it later this week / early next

**Comment by bring728** [comment]
@krrish94 I am also curious about this. How did it go?
